svir/help/source/04_irmt_settings.rst

```diff
@@ -40,6 +40,15 @@ service is hosted. By default, the host is set to `http://localhost:8800`,
 meaning that the plugin will attempt to connect to a OQ Engine Server running
 locally, on port `8800`.
 
+When the plugin loads a map from an OQ-Engine's output or when integrated risk
```

Comment:
Not sure if we should use a `'s` here, we should an English speaker. I thought it was used only for persons, not for objects.